### ISSUE TITLE
Consider using torch.nn.KLDivLoss

### ISSUE THREAD

**Issue by jkelling** [Photon-AI-Research/InSituML#124]
Consider replacing [own KL loss implementation](https://github.com/Photon-AI-Research/InSituML/blob/khi/main/ModelHelpers/cINN/train_khi_AE_refactored/utilities.py#L90) by [torch's](https://pytorch.org/docs/stable/generated/torch.nn.KLDivLoss.html).
